src/haddock/libs/libcns.py

```diff
@@ -117,42 +126,42 @@ def write_eval_line(param, value, eval_line='eval (${}={})'):
         return eval_line.format(param, value)
 
     else:
-        emsg = f'Unexpected type when writing CNS header: {type(value)}'
+        emsg = f"Unexpected type when writing CNS header: {type(value)}"
         log.error(emsg)
         raise TypeError(emsg)
 
 
 def load_ff_parameters(forcefield_parameters):
     """Add force-field specific parameters to its appropriate places."""
-    ff_param_header = f'{linesep}! FF parameters{linesep}'
-    ff_param_header += f'parameter{linesep}'
-    ff_param_header += f'  @@{forcefield_parameters}{linesep}'
-    ff_param_header += f'end{linesep}'
+    ff_param_header = f"{linesep}! FF parameters{linesep}"
+    ff_param_header += f"parameter{linesep}"
+    ff_param_header += f"  @@{forcefield_parameters}{linesep}"
+    ff_param_header += f"end{linesep}"
 
     return ff_param_header
 
 
 def load_ff_topology(forcefield_topology):
     """Add force-field specific topology to its appropriate places."""
-    ff_top_header = f'{linesep}! Toplogy{linesep}'
-    ff_top_header += f'topology{linesep}'
-    ff_top_header += f'  @@{forcefield_topology}{linesep}'
-    ff_top_header += f'end{linesep}'
+    ff_top_header = f"{linesep}! Toplogy{linesep}"
+    ff_top_header += f"topology{linesep}"
```

Comment:
Same comment as before - the only top file required to be passed to the CNS input files is for ligands if present.

Reply:
removed!